src/boot/efi/boot.c

```diff
@@ -1596,6 +1608,8 @@ static void config_load_defaults(Config *config, EFI_FILE *root_dir) {
                 .editor = true,
                 .auto_entries = true,
                 .auto_firmware = true,
+                .auto_shutdown = true,
+                .auto_reboot = true,
```

Comment:
i would prefer if we leave this off, so that people can enable it if they want.

(ideally we'd auto-enable it if we would know that the kbd is too limited for character input, but afaics uefi has no api for detecting that)